src/containers/App/navigation.ts

```diff
@@ -0,0 +1,29 @@
+import { buildPath } from '../shared/routing'
```

Comment:
There's no code change in this file, right (just moving)? I did a quick skim and it doesn't look like it, but I figured I should confirm

Reply:
This is a new file.  Its contents came from routes.ts to prevent circular dependencies caused by import routes into the Search component.